### ISSUE TITLE
conversion error

### ISSUE THREAD

**Issue by SebGue** [Eomys/SciDataTool#4]
Hello Helene,

there is an other issue with the fft/ifft regarding the constant part of the data which is not converted correctly.
You can reproduce this issue with the follow code:

```
from SciDataTool import Data1D, DataTime
from numpy import linspace, ones_like

time = linspace(0, 1, 8, endpoint=False)    
field = ones_like(time)

Time = Data1D(name="time", unit="s", values=time)
y = DataTime(name="TestData",axes=[Time],values=field)

y_ = y.time_to_freq().freq_to_time()

```

Best regards, Sebastian

**Comment by helene-t** [comment]
Hello Sebastian,

I corrected the normalization for the inverse fft in the latest release (1.0.6). Can you tell me if it fixes your issue?

Helene

**Comment by SebGue** [comment]
Hello Helene,

thank you for your affort. The constant part of the data is converted correctly now.
But the harmonics now convert to twice its original value.
You can check this behavior with the following field:
`field = cos(2*pi*time) + 1`

I think this simple case is worth an unitest. What do you think?
Best regards, Sebastian

**Comment by helene-t** [comment]
You are right, sorry about that. I corrected this issue in the release 1.0.7.

By the way, if you want to test a feature you just developed in another software which uses SciDataTool as a package, you can upload your version to [Test PyPi](https://test.pypi.org/) (first you'll need to create an account), without creating a release on GitHub, using the following procedure:

```
python setup.py sdist bdist_wheel
twine upload --repository testpypi dist/*
pip install --index-url https://test.pypi.org/simple/ SciDataTool
```

Best regards, Helene

**Comment by SebGue** [comment]
Thanks alot, now the values are converted correctly.

Regrading the test of PyPi, you should add this to the documentation. It is a great feature worth mentioning.
